java/src/jmri/jmrix/dccpp/DCCppCommandStation.java

```diff
@@ -268,6 +268,19 @@ public boolean isProgramV4Supported() {
         return ret;  
     }
 
+    /**
+     * Can this command station handle the <#> request to get supported number of cabs(locos)?
```

Comment:
Javadoc cannot handle `<#>` so I suggest you change it to `&lt;#&gt;`

Static analysis failed with:
```
Error:  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.1.0:javadoc (default-cli) on project jmri: An error has occurred in Javadoc report generation: 
Error:  Exit code: 1 - /home/runner/work/JMRI/JMRI/java/src/jmri/jmrix/dccpp/DCCppCommandStation.java:272: error: malformed HTML
Error:       * Can this command station handle the <#> request to get supported number of cabs(locos)?
Error:                                             ^
Error:  /home/runner/work/JMRI/JMRI/java/src/jmri/jmrix/dccpp/DCCppCommandStation.java:272: error: bad use of '>'
Error:       * Can this command station handle the <#> request to get supported number of cabs(locos)?
```